src/main/java/it/gov/pagopa/pu/workflow/wf/classification/transfer/wfclassification/TransferClassificationWFImpl.java

```diff
@@ -31,9 +36,24 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
   }
 
   @Override
-  public void classify(Long orgId, String iuv, String iur, int transferIndex) {
-    log.info("Handling Transfer classification for organization id {} and iuv {} and iur {} and transfer index {}", orgId, iuv, iur, transferIndex);
-    transferClassificationActivity.classify(new TransferSemanticKeyDTO(orgId, iuv, iur, transferIndex));
-    log.info("Ingestion organization id {} and iuv {} and iur {} and transfer index {} is completed", orgId, iuv, iur, transferIndex);
+  public void classify() {
+    toClassify.stream().distinct()
+      .forEach(item -> {
+        log.info("Handling Transfer classification for semantic key", item);
```

Comment:
if you don't place a placeholder, it will not print the argument
```suggestion
        log.info("Handling Transfer classification for semantic key {}", item);
```